src/context/WalletProvider/NewWalletViews/components/PairBody.tsx

```diff
@@ -0,0 +1,61 @@
+import { Alert, AlertDescription, AlertIcon, Button, Flex, Spinner } from '@chakra-ui/react'
+import type { InterpolationOptions } from 'node-polyglot'
+import type { ReactNode } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { Text } from 'components/Text'
+
+const spinner = <Spinner color='white' />
+
+type PairBodyProps = {
+  icon: ReactNode
+  headerText: string | [string, InterpolationOptions]
+  bodyText: string | [string, InterpolationOptions]
+  buttonText: string
```

Comment:
Maybe translation vernacular to make it more clear that these aren't raw text:

```suggestion
  headerTranslation: string | [string, InterpolationOptions]
  bodyTranslation: string | [string, InterpolationOptions]
  buttonTranslation: string
```

Reply:
1509cd984ad79b48bdd814f19df97024b01995e1 here she is!